setup.py

```diff
@@ -35,10 +34,8 @@
     packages=find_packages(),
     install_requires=[
         'Django>=1.8,<1.10',
-        # 0.3.0 introduced a major breaking change, effectively gutting PSA and
-        # moving its Django components to a new social-auth-app-django package.
-        # We may require the new package at some point in the future, once it's stable.
-        'python-social-auth>=0.2.21,<0.3.0',
-        'six>=1.10.0,<2.0.0'
+        'six',
+        'social-auth-core[openidconnect]',
+        'social-auth-app-django',
```

Comment:
I think the experience we've had here demonstrates the value of at least pinning to a minor version for these packages.